templates/index.html

```diff
@@ -65,25 +67,25 @@ <h3 class="p-heading--4">Comprehensive cloud platform for distributed low-latenc
           <p>MicroStack answers the needs of edge computing, providing a secure, reliable and scalable cloud platform with minimal footprint and simplified lifecycle management capabilities.</p>
           <p>All to meet the requirements of the most demanding applications in telecom, industrial, automotive and other market sectors that need edge infrastructure.</p>
           <p>
-            <a href="https://ubuntu.com/blog/edge-computing-is-dead-long-live-micro-clouds-and-iot-gateways" class="p-link--external">Read a blog about Canonical's edge strategy</a>
+            <a href="https://ubuntu.com/blog/edge-computing-is-dead-long-live-micro-clouds-and-iot-gateways" >Read a blog about Canonical's edge strategy</a>
           </p>
         </div>
-        <div class="col-6 u-align--center u-vertically-center">
-          <img src="https://assets.ubuntu.com/v1/dcb2963c-openstack+cloud+outlines.svg" class="u-hide--small" style="height: 225px; width: 394px;" alt="openstack cloud outlines">
+        <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
+          <img src="https://assets.ubuntu.com/v1/dcb2963c-openstack+cloud+outlines.svg" style="height: 225px; width: 394px;" alt="openstack cloud outlines">
```

Comment:
Not part of the upgrade, but we could update these as a drive by.
```suggestion
          <img src="https://assets.ubuntu.com/v1/dcb2963c-openstack+cloud+outlines.svg" style="height: 225px; width: 394px;" alt="">
```